vizro-core/src/vizro/static/css/loading.css

```diff
@@ -0,0 +1,60 @@
+/* Loading spinner */
+._dash-loading {
+  background-color: var(--root-loading-color);
+  inset: 0;
+  color: transparent;
+  margin: 0;
+  position: fixed;
+  text-align: center;
+}
+
+._dash-loading::after {
+  animation:
+    load6 1.7s infinite ease,
+    round 1.7s infinite ease;
+  border-radius: 50%;
+  color: var(--root-loading-contrast);
+  content: "";
+  display: inline-block;
+  font-size: 60px;
+  height: 1em;
+  margin: 2rem auto 0;
+  vertical-align: text-bottom;
+  width: 1em;
+}
+
+@keyframes load6 {
+  0%,
+  5%,
+  95%,
+  100% {
+    box-shadow:
+      0 -0.83em 0 -0.4em,
+      0 -0.83em 0 -0.42em,
+      0 -0.83em 0 -0.44em,
+      0 -0.83em 0 -0.46em,
+      0 -0.83em 0 -0.477em;
+  }
+
+  10%,
+  59% {
+    box-shadow:
+      0 -0.83em 0 -0.4em,
+      -0.087em -0.825em 0 -0.42em,
+      -0.173em -0.812em 0 -0.44em,
+      -0.256em -0.789em 0 -0.46em,
+      -0.297em -0.775em 0 -0.477em;
+  }
+
+  /* Other keyframes omitted for brevity */
```

Comment:
Is this omitted just from the example but we should have it? Or we don't actually want it? 🤔

Reply:
I used two suggestions by ChatGPT to shorten the CSS here a bit:

> @-webkit-keyframes load6 and @keyframes load6 are identical, as well as @-webkit-keyframes round and @keyframes round. Since modern browsers support non-prefixed @keyframes, you might not need the -webkit- prefix, unless you must support very old versions of Safari or Android browsers.

I reckoned that this was a reasonable suggestion, and the resulting CSS looked much more neat and digestible.